app/frontend/styles/provider/_rejection.scss

```diff
@@ -3,6 +3,18 @@
   padding-right: 70px;
   position: relative;
 
+  p {
```

Comment:
Couldn't use `last-of-type` or `last-child` here because the change link when the component is editable is the last `p` in the block.

Reply:
(This is also copied from prototype styling here: https://github.com/DFE-Digital/manage-teacher-training-applications-prototype/blob/8bb93227ebd80d8aeb80e5643e26461bfade765b/app/views/_components/feedback/_feedback.scss)